pkg/terraform/presets.go

```diff
@@ -22,6 +23,9 @@ func createPresetValues(b *Block) map[string]cty.Value {
 	switch b.TypeLabel() {
 	case "aws_iam_policy_document":
 		presets["json"] = cty.StringVal(b.ID())
+	// If the user leaves the name blank, Terraform will automatically generate a unique name
+	case "aws_launch_template":
+		presets["name"] = cty.StringVal(uuid.New().String())
```

Comment:
do we have to use a UUID here?

Reply:
We need to generate some unique value in case the user did not pass the `name` argument.